src/less/index.css

```diff
@@ -64,6 +65,7 @@
   float: right;
   color: white;
   margin-left: 5px;
+  font-family: moon-regular, Arial, sans-serif;
```

Comment:
Lots of these fonts :/ maybe we should create some global class and extend if needed ? I think that we need consistency in fonts.